src/public/javascripts/app.js

```diff
@@ -7,7 +7,7 @@ Chart.defaults.global.elements.line.backgroundColor = "rgba(0,0,0,0)";
 Chart.defaults.global.elements.line.borderColor = "rgba(0,0,0,0.9)";
 Chart.defaults.global.elements.line.borderWidth = 2;
 
-var socket = io(location.protocol + '//' + location.hostname + ':' + location.port);
+var socket = io(location.protocol + '//' + location.hostname + ':' + (port ? port : location.port));
 var defaultSpan = 0;
```

Comment:
as per my comment before here we should be able to get the port value saved against some `data-*` attribute